src/Responses/FineTuning/RetrieveJobResponse.php

```diff
@@ -66,6 +67,7 @@ public static function from(array $attributes, MetaInformation $meta): self
             $attributes['validation_file'],
             $attributes['training_file'],
             $attributes['trained_tokens'],
+            RetrieveJobResponseError::from($attributes['error']),
```

Comment:
```suggestion
            isset($attributes['error']) ? RetrieveJobResponseError::from($attributes['error']) : null,
```

Reply:
i think  `RetrieveJobResponseError::from($attributes['error'] ?? null)` is better